library/common/main_interface.cc

```diff
@@ -16,6 +16,9 @@
 #include "library/common/buffer/utility.h"
 #include "library/common/http/dispatcher.h"
 #include "library/common/http/header_utility.h"
+#include "library/common/include/config_template.h"
+
+char* config_template() { return strdup(config_template_str); }
```

Comment:
Make the return type const and you can drop the `strdup`.

Reply:
Actually, better yet, let's drop this function altogether, and just put the `extern`'d const on the `main_interface.h`. No real benefit to indirection here.

Reply:
You can still define the `const` in a separate file.

Reply:
Done